[13.0][MIG] purchase_order_line_price_history: Migration to 13.0

ernestotejeda opened the pull request:
cc @Tecnativa TT24923

Commit:
[ADD] purchase_order_line_price_history: new module

oca-travis committed:
[UPD] Update purchase_order_line_price_history.pot

OCA-git-bot committed:
[UPD] README.rst

Commit:
[FIX] purchase_order_line_price_history: get line product

OCA-git-bot committed:
purchase_order_line_price_history 12.0.1.0.1

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: purchase-workflow-12.0/purchase-workflow-12.0-purchase_order_line_price_history
Translate-URL: https://translation.odoo-community.org/projects/purchase-workflow-12-0/purchase-workflow-12-0-purchase_order_line_price_history/

Commit:
[FIX] purchase_order_line_price_history: fix set price.

OCA-git-bot committed:
purchase_order_line_price_history 12.0.1.0.2

Commit:
[IMP] purchase_order_line_price_history: black, isort, prettier

Commit:
[MIG] purchase_order_line_price_history: Migration to 13.0

HviorForgeFlow reviewed (commented):
Functional Review: Works as expected

Could be feasible to move the set price button to the Purchase history tree view? Or at least drop the default Save ... Close buttons on the History Lines popup.

CarlosRoca13 reviewed (approved):
Functional and code review :+1:

ernestotejeda commented:
Thanks @HviorForgeFlow, I have dropped the default Save buttons on the History Lines popup. 
And I also added the hook method added here in v12 https://github.com/OCA/purchase-workflow/pull/953

Tardo commented on purchase_order_line_price_history/wizards/purchase_order_line_price_history.py:
```diff
@@ -0,0 +1,95 @@
+        vals = []
+        order_lines = self.env["purchase.order.line"].search(domain, limit=20)
+        order_lines -= self.purchase_order_line_id
+        for order_line in order_lines:
+            vals.append((0, False, {"purchase_order_line_id": order_line.id}))
```
```suggestion
        vals = [(0, False, {"purchase_order_line_id": order_line.id}) for order_line in order_lines]
```

Tardo reviewed (approved):
:+1: LGTM. Code review... only non-blocking comments.

ernestotejeda commented:
@Tardo Changes done
Sometimes I think that in some cases it is more readable if we don't use list comprehension, but it is certainly a guideline to use them, so let use them. Thanks.

HviorForgeFlow commented:
/ocabot merge nobump
